src/pages/admin/quizzes/QuizEdit/index.js

```diff
@@ -48,7 +83,65 @@ const QuizEdit = () => {
             <Button className={style.sidebarButtons}>Add a Question</Button>
             <Button className={style.sidebarButtons}>Change Category</Button>
           </div>
-          {/*************** INSERT <QUESTION FORM> CODE HERE ***************/}
+          <div>
+            {questionType === 'multiple_choice' ? (
+              <MultipleChoiceType questions={questions[0]} />
+            ) : (
+              <IdentificationType questions={questions[1]} />
+            )}
```

Comment:
maybe we can put this in a function and use switch case again. if ever we need to add more choices, it will be easier